JournalCoreData/JournalCoreData/Extensions/String+Extension.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  String+Extension.swift
+//  JournalCoreData
+//
+//  Created by Nelson Gonzalez on 2/11/19.
+//  Copyright © 2019 Nelson Gonzalez. All rights reserved.
+//
+
+import UIKit
+
+extension String {
+    
+    static func dateToString(date: Date) -> String{
+        
+        // formatter #1
+        let formatter1 = DateFormatter()
+        formatter1.dateFormat = "MM-dd-yyyy"
```

Comment:
You could use default styles and do everything without needing a second `DateFormatter`
```suggestion
        formatter1.dateStyle = .short
        formatter1.timeStyle = .short
```